src/components/App/App.js

```diff
@@ -116,7 +116,9 @@ export default class App extends React.Component {
     const comboCount = this.state.msgCount[this.Authentication.getUserId()] || 0;
     if (this.state.finishedLoading && this.state.isVisible) {
       return (
-        <h1 style={{ fontSize: '100px', color: "white" }}>{`COMBO: ${comboCount}`}</h1>
+        <div className="circleContainer" style={{zIndex: '1', borderRadius:'50%', width: '100px', height: '100px', background: 'lightblue'}}>
```

Comment:
non-blocking: Should we create a react component for this (like `ComboCount` component)? As we add more elements in it could help readability.

nitpick: Do you plan to use the `circleContainer`class for anything, or are you gonna stick with inline styles?
either way is fine, but if the class name won't be used, we can just remove it.

Reply:
padding-bottom on the wrapping div that is circleContainer seems to do the trick, will update!
the inline-styling will also move into the CSS, for now, it will be within the app but can be moved into its own custom CSS